metasv/generate_final_vcf.py

```diff
@@ -92,6 +92,13 @@ def get_interval_info(feature,pass_calls):
         if "AS" in svmethods:
             pos, end, svlen = map(int, feature.fields[6:9])
             is_pass = svlen >= 100               
+    elif "DUP" in sub_types:
+        index_to_use = sub_types.index("DUP")
```

Comment:
Can factor out the common calculations for each svtype instead of having the long chain of `if ... elif`. A for-loop would be good enough to capture the priority of svtypes we have here. Also, since AS is not supported for CTX, ITX, the check `if "AS" in svmethods` would fail for them so pretty much the code for each svtype is the same (except for INS).